desc/objectives/objective_funs.py

```diff
@@ -685,48 +692,10 @@ def weights(self):
             [jnp.ones(obj.dim_f) * obj.weight for obj in self.objectives]
         )
 
-    @property
-    def _all_things(self):
-        """list: all things known to this objective, used and unused."""
-        if not hasattr(self, "_extra_things"):
-            self._extra_things = []
-        return [obj.things for obj in self.objectives] + self._extra_things
-
     @property
     def things(self):
```

Comment:
maybe don't need a public attribute for this?

Reply:
We probably don't _need_ it, but we have it public for `Optimizable` and I don't think it's hurting anything.

Reply:
Not sure what you mean? `things` is only defined for objectives. My point was that it's not something we necessarily want the user messing with.

Reply:
Oh sorry, scratch what I said about `Optimizeable`. But this is the `ObjectiveFunction` class, and it is an important attribute. This is only a getter method -- I removed the setter method -- so the users shouldn't be able to mess with it.